src/main/java/me/dsnet/cetriolo/antlr/output/GherkinParser.java

```diff
@@ -1,2552 +0,0 @@
-import java.util.List;
-import java.util.ArrayList;
-
-import org.antlr.runtime.tree.*;
-
-
-@SuppressWarnings({"all", "warnings", "unchecked"})
-public class GherkinParser extends Parser {
-    public static final String[] tokenNames = new String[] {
-        "<invalid>", "<EOR>", "<DOWN>", "<UP>", "BACKGROUND", "COMMENT", "DOCSTR", "EXAMPLE", "FEATURE", "FEAT_KEY", "NL", "NUMBER", "PHOLDER", "SCENARIO", "SCEN_OUT", "STEP_KEY", "STRING", "TAGNAME", "T_ROW", "WORD", "WS"
-    };
-
-    public static final int EOF=-1;
-    public static final int BACKGROUND=4;
-    public static final int COMMENT=5;
-    public static final int DOCSTR=6;
-    public static final int EXAMPLE=7;
-    public static final int FEATURE=8;
-    public static final int FEAT_KEY=9;
-    public static final int NL=10;
-    public static final int NUMBER=11;
-    public static final int PHOLDER=12;
-    public static final int SCENARIO=13;
-    public static final int SCEN_OUT=14;
-    public static final int STEP_KEY=15;
-    public static final int STRING=16;
-    public static final int TAGNAME=17;
-    public static final int T_ROW=18;
-    public static final int WORD=19;
-    public static final int WS=20;
-
-    // delegates
-    public Parser[] getDelegates() {
-        return new Parser[] {};
-    }
-
-    // delegators
-
-
-    public GherkinParser(TokenStream input) {
-        this(input, new RecognizerSharedState());
-    }
-    public GherkinParser(TokenStream input, RecognizerSharedState state) {
-        super(input, state);
-    }
-
-protected TreeAdaptor adaptor = new CommonTreeAdaptor();
-
-public void setTreeAdaptor(TreeAdaptor adaptor) {
-    this.adaptor = adaptor;
-}
-public TreeAdaptor getTreeAdaptor() {
-    return adaptor;
-}
-    public String[] getTokenNames() { return GherkinParser.tokenNames; }
-    public String getGrammarFileName() { return "C:\\Users\\sessonad\\Documents\\NetBeansProjects\\Cetriolo\\src\\me\\dsnet\\cetriolo\\antlr\\Gherkin.g"; }
-
-
-    	public List<SyntaxError> syntaxErrors = new ArrayList<SyntaxError>();
-
-        @Override
-        public String getErrorMessage(RecognitionException ex, String[] tokenNames) {
-            String message = super.getErrorMessage(ex, tokenNames);
-            SyntaxError.ErrorType type= null;
-            StackTraceElement firstelem = ex.getStackTrace()[0];
-            String methodName= firstelem.getMethodName();
-            ex.printStackTrace();
-            
-            
-            if(ex instanceof NoViableAltException){
-                NoViableAltException e = (NoViableAltException)ex;
-                String grammarDesc = e.grammarDecisionDescription;
-                if(grammarDesc.contains("scenario | scenario_outline")){
-                    type=SyntaxError.ErrorType.NOT_VIABLE_SCENARIO;
-                }else if(grammarDesc.contains(" step ")){
-                    type=SyntaxError.ErrorType.NOT_VIABLE_FEATURE;
-                }else if(grammarDesc.contains(" background ")){
```

Comment:
⚠️ This specific condition did not exist in the grammar, so it's missing from the new generated code. Please let me know if I need to update the grammar to use it.

Reply:
Oh yes, that's true :).